packages/oidc-client/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@actions/oidc-client",
```

Comment:
I am not sure whether we should create a new package for OIDC which currently only have a single method `getIDToken()`

It might make more sense to create a new package if we are going to have a set of methods around OIDC.